src/common/messages.ts

```diff
@@ -238,6 +238,7 @@ export const Messages = {
         ExpandAllRules: `${messagePrefix}/assessmentCardSelection/expandAllRules`,
         ToggleVisualHelper: `${messagePrefix}/assessmentCardSelection/toggleVisualHelper`,
         NavigateToNewCardsView: `${messagePrefix}/assessmentCardSelection/navigateToNewCardsView`,
+        AssessmentStoreChanged: `${messagePrefix}/assessmentCardSelection/assessmentStoreChanged`,
```

Comment:
I believe we will want to have 2 assessment card controllers eventually- one for assessment data and one for quick assess data. As such, we will have to duplicate a lot of this code including the following:

- Add an additional AssessmentStoreChanged message to the QuickAssesCardSelection section
- Mirror the logic that this PR is adding in the assessment-card-selection-action-creator in the quick-assess-card-selection-action-creator
- Initialize the controller twice- once with the assessmentCardSelectionMessageCreator and once with the quickAssessCardSelectionMessageCreator

If you prefer, I'm okay with putting this off until another PR though. I forgot to make the message creator able to handle both assessment and quick assess messages in the last PR I made, so those changes are not yet in main for you to integrate with, which would be a blocker. Up to you if you prefer to wait until my next PR is merged or if you want to just port that change to this PR. Here's the necessary change: https://github.com/microsoft/accessibility-insights-web/pull/6501/files#diff-63c0ad049698d3b0edac710cab5bfa621ff664673990ab907f5280ca8d0bb5ceL11-L90